src/invidious/videos.cr

```diff
@@ -190,7 +190,7 @@ VIDEO_FORMATS = {
   "248" => {"ext" => "webm", "height" => 1080, "format" => "DASH video", "vcodec" => "vp9"},
   "271" => {"ext" => "webm", "height" => 1440, "format" => "DASH video", "vcodec" => "vp9"},
   # itag 272 videos are either 3840x2160 (e.g. RtoitU2A-3E) or 7680x4320 (sLprVF6d7Ug)
-  "272" => {"ext" => "webm", "height" => 2160, "format" => "DASH video", "vcodec" => "vp9"},
```

Comment:
Why removing the height?

Reply:
Oops, that was a remnant of another fix I tried to overload into this PR.